clients/src/test/java/org/apache/kafka/clients/producer/internals/TransactionManagerTest.java

```diff
@@ -1678,6 +1678,39 @@ public void testMultipleAddPartitionsPerForOneProduce() throws InterruptedExcept
         assertTrue(secondResponseFuture.isDone());
     }
 
+    @Test
+    public void testRetriableErrors() {
```

Comment:
Is it possible to parameterize this test case so that we can verify retriable errors other than REQUEST_TIMED_OUT (e.g. CONCURRENT_TRANSACTIONS).

Reply:
We can, but do you want to test all of them? or just the ones that can be returned?

Reply:
Just the ones we expect. So `REQUEST_TIMED_OUT`, `CONCURRENT_TRANSACTIONS` and any of the others we've replaced with the implicit `RetriableException` check.